### DISCUSSION TITLE
Github webhook response is BROKEN!

### DISCUSSION THREAD

**Discussion by yunusemreuveyik** [community/community#74630] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/16350742/1eed3f48-c2a6-4751-89ce-16bf426c1ae4)

![image](https://github.com/community/community/assets/16350742/10f9b241-03ab-411e-88ad-7b130383ff60)

**Comment by ghostinhershell** [comment]
Hi @yunusemreuveyik, 

Thanks so much for reaching out in the community. Is this issue resolved or have you found the answer to your question here?
